WSL/wsl-config.md

```diff
@@ -247,15 +247,15 @@ Section label: `[experimental]`
 
 | Setting name | Value | Default | Notes |
 |:----|:----|:----|:----|
-|`useWindowsDnsCache`**| bool | false | Only applicable when `experimental.dnsTunneling` is set to true. When this option is set to True, DNS requests tunneled from Linux will bypass cached names within Windows to always put the requests on the wire. |
+|`useWindowsDnsCache`**| bool | false | Only applicable when `experimental.dnsTunneling` is set to true. When this option is set to false, DNS requests tunneled from Linux will bypass cached names within Windows to always put the requests on the wire. |
```

Comment:
@craigloewen-msft Can you confirm?

Reply:
Ok - confirmed this is correct and that we had this backwards. Thank you for the correction @zfb132!